tenant/src/test/scala/com/improving/app/tenant/TestData.scala

```diff
@@ -1,29 +1,58 @@
 package com.improving.app.tenant
 
-import com.improving.app.common.domain.{Address, CaPostalCodeImpl, Contact, PostalCodeMessageImpl}
-import com.improving.app.tenant.domain.{TenantInfo, TenantOrganizationList}
+import com.improving.app.common.domain.{
+  Address,
+  CaPostalCodeImpl,
+  Contact,
+  EditableAddress,
+  EditableContact,
+  PostalCodeMessageImpl
+}
+import com.improving.app.tenant.domain.{EditableTenantInfo, TenantInfo, TenantOrganizationList}
 
 object TestData {
-  val baseContact = Contact(
-    firstName = "firstName",
-    lastName = "lastName",
+  val baseContact: EditableContact = EditableContact(
+    firstName = Some("firstName"),
+    lastName = Some("lastName"),
     emailAddress = Some("test@test.com"),
     phone = Some("111-111-1111"),
-    userName = "contactUsername"
+    userName = Some("contactUsername")
   )
-  val baseAddress = Address(
-    line1 = "line1",
-    line2 = "line2",
-    city = "city",
-    stateProvince = "stateProvince",
-    country = "country",
+  val baseAddress: EditableAddress = EditableAddress(
+    line1 = Some("line1"),
+    line2 = Some("line2"),
+    city = Some("city"),
+    stateProvince = Some("stateProvince"),
+    country = Some("country"),
     postalCode = Some(PostalCodeMessageImpl(CaPostalCodeImpl("caPostalCode")))
   )
 
-  val baseTenantInfo = TenantInfo(
-    name = "Tenant Name",
-    primaryContact = Some(baseContact),
-    address = Some(baseAddress),
-    organizations = Some(TenantOrganizationList())
+  def infoFromEditableInfo(editable: EditableTenantInfo): TenantInfo = TenantInfo(
```

Comment:
This seems like it should be composed from contactFromEditableContact and addressFromEditableAddress methods which we will probably want down the road

Reply:
come to think of it, it'd be a good idea to add code like this to the implementation side, really

Reply:
In tests we don't need to do our calls to check that the editable fields aren't `None`. However in our service the same copying we do in EditTenantInfo with said checks will need to happen on transition out of `DraftState` - once I make the change to put State objects in protobufs and use `EditableInfo` in `DraftState`. So I will break out the service function as well

Reply:
I think one function, `EditableFoo => Either[Error, Foo]` and one function `(Foo, EditableFoo) => Foo` for each of these message types should be good. the latter one might need to return an `Either` too, not sure

Reply:
I don't think a higher order function is needed here, just one for each message type